static/app/views/replays/detail/domMutations/index.tsx

```diff
@@ -148,13 +149,14 @@ const CodeContainer = styled('div')`
   overflow: auto;
   max-height: 400px;
   max-width: 100%;
+  z-index: 1;
 `;
 
 const StepConnector = styled('div')`
   position: absolute;
   height: 100%;
   top: 28px;
-  left: 31px;
+  left: 28px;
   border-right: 1px ${p => p.theme.border} dashed;
 `;
```

Comment:
The CSS for this has other issues. But that's not part of the ticket or test plan, so we really don't need to change any of it.

Reply:
What's specifically wrong with it? I ripped that one from somewhere else in the code base since I was just trying to prototype something that looked decent initially since there wasn't a design